index.js

```diff
@@ -58,7 +58,7 @@ var defaultPluralRules = {
     bosnian_serbian: russianPluralGroups,
     chinese: function () { return 0; },
     croatian: russianPluralGroups,
-    french: function (n) { return n > 1 ? 1 : 0; },
+    french: function (n) { return n >= 2 ? 1 : 0; },
```

Comment:
`> 1` and `>= 2` are identical for integers. is there a need to change this?

Reply:
1.99 is singular in french. 2 is plural. ☺️

Reply:
Yup, this is to handle non-integers between 1 and 2 @ljharb

Reply:
This api only expects integers.

Reply:
Well this is unexpected, should I have a go at it or this is unwanted ?

Reply:
No, this seems a fine fix, even if it’s not needed for the integer case. I’ll review it tomorrow.

Reply:
Alright, thanks !